sphinx_rtd_theme/theme.conf

```diff
@@ -1,7 +1,7 @@
 [theme]
 inherit = basic
 stylesheet = css/theme.css
-pygments_style = default
+pygments_style = sphinx
```

Comment:
This is an unintended change please revert this.

Reply:
@Blendify, you are right! I revert this change :+1: thanks for your feedback :nerd_face: